clients/src/test/java/org/apache/kafka/clients/MetadataTest.java

```diff
@@ -1185,9 +1263,14 @@ else if (partition.equals(internalPart))
     /**
      * For testUpdatePartially, validates that updatedMetadata is matching expected part1Metadata, part2Metadata, interalPartMetadata, nodes & more.
      */
-    void validateForUpdatePartitionLeadership(Metadata updatedMetadata, PartitionMetadata part1Metadata, PartitionMetadata part2Metadata, PartitionMetadata internalPartMetadata,
-        List<Node> expectedNodes, String expectedClusterId, Set<String> expectedUnauthorisedTopics, Set<String> expectedInvalidTopics, Set<String> expectedInternalTopics,
-        Node expectedController, Map<String, Uuid> expectedTopicIds) {
+    void validateForUpdatePartitionLeadership(Metadata updatedMetadata,
+                                              PartitionMetadata part1Metadata, PartitionMetadata part2Metadata, PartitionMetadata part12Metadata,
```

Comment:
nit: (for refactor) -- if we decided to put all the arguments on new lines, let's do it for the all the arguments. The ones with three are a bit harder to read.

Reply:
thanks - would you mind leaving this for the refactor PR?

Reply:
yup that's fine.